src/Model/FilterJob.re

```diff
@@ -83,6 +83,7 @@ module Make = (Config: Config) => {
     let currentMatches = ListEx.firstk(Constants.maxItemsToFilter, filtered);
 
     // If the new query matches the old one... we can re-use results
+    // TODO: This breaks new queries! The letter highlights will be wrong.
```

Comment:
Not to do with me that I can see, but searching for say `vecto` and letting it complete then adding `r` to make `vector` will show the results for `vector` but with `vecto` highlighted.

Reply:
I wonder if `fzy` is fast enough that we could just scrap this whole block?
```
    if (pending.filter != ""
        && Filter.fuzzyMatches(oldExplodedFilter, filter)
        && List.length(currentMatches) < Constants.maxItemsToFilter) {
...
}
```

It's really just an optimization when the query is refined to not have a jarring set of results, but it might not be necessary anymore.